packages/bits/demo/src/components/demo/sorter/sorter-test/sorter-test.example.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, OnInit, ViewChild } from "@angular/core";
```

Comment:
These files dont need to be added. I just kept them in order to make verification earlier.